include/zephyr/llext/llext.h

```diff
@@ -202,11 +203,15 @@ int llext_add_domain(struct llext *ext, struct k_mem_domain *domain);
  * or object.
  *
  * @param[in] rel Relocation data provided by elf
- * @param[in] opaddr Address of operation to rewrite with relocation
- * @param[in] opval Value of looked up symbol to relocate
+ * @param[in] loc Address of operation to rewrite with relocation
+ * @param[in] sym_base_addr Symbol address
+ * @param[in] sym_name Symbol name
+ * @param[in] load_bias .text load address
+ * @retval 0 success
+ * @retval -ENOEXEC invalid relocation
  */
-void arch_elf_relocate(elf_rela_t *rel, uintptr_t opaddr, uintptr_t opval);
-
+int32_t arch_elf_relocate(elf_rela_t *rel, uintptr_t loc,
```

Comment:
Error return values use generic `int` everywhere else. Please replace in `llext.c` and `elf.c` as well.
```suggestion
int arch_elf_relocate(elf_rela_t *rel, uintptr_t loc,
```